src/main/scala/firrtl/passes/wiring/WiringTransform.scala

```diff
@@ -39,7 +39,8 @@ case class SinkAnnotation(target: Named, pin: String) extends SingleTargetAnnota
 class WiringTransform extends Transform with DependencyAPIMigration {
 
   override def prerequisites = Forms.MidForm
-  override def optionalPrerequisites = Seq.empty
+  // once wire targets are turned into nodes, our logic to wire them up no longer works
+  override def optionalPrerequisites = Seq(Dependency[firrtl.transforms.RemoveWires])
   override def optionalPrerequisiteOf = Forms.MidEmitters
```

Comment:
I think this is supposed to be added to `optionalPrerequisiteOf` and not `optionalPrerequisites`? (The idea is to inject a prerequisite of `WiringTransform` into `RemoveWires` so that the `WiringTransform` runs before `RemoveWires`, right?)

Reply:
oh yeah, sorry

Reply:
(surprised that all our tests still pass)